codec-http/src/test/java/io/netty/handler/codec/http/HttpResponseDecoderTest.java

```diff
@@ -369,6 +369,29 @@ public void testLastResponseWithHeaderRemoveTrailingSpaces() {
     }
 
     @Test
+    public void testResetContentResponseWithTransferEncoding() {
+        EmbeddedChannel ch = new EmbeddedChannel(new HttpResponseDecoder());
+        ch.writeInbound(Unpooled.copiedBuffer(
+                "HTTP/1.1 205 Reset Content\r\n" +
+                "Transfer-Encoding: chunked\r\n" +
+                "\r\n" +
+                "0\r\n" +
+                "\r\n",
+                CharsetUtil.US_ASCII));
+
+        HttpResponse res = ch.readInbound();
+        assertThat(res.protocolVersion(), sameInstance(HttpVersion.HTTP_1_1));
+        assertThat(res.status(), is(HttpResponseStatus.RESET_CONTENT));
+
+        LastHttpContent lastContent = ch.readInbound();
+        assertThat(lastContent.content().isReadable(), is(false));
+        lastContent.release();
+
+        assertThat(ch.finish(), is(false));
+        assertThat(ch.readInbound(), is(nullValue()));
```

Comment:
you can remove this line as if `ch.finish()` returns false it means there is nothing in the inbound and outbound queue.

Reply:
done